templates/openstack/install.html

```diff
@@ -41,56 +41,52 @@ <h1>Install OpenStack yourself</h1>
+        <li class="p-list__item is-ticked">Core services included</li>
+        <li class="p-list__item is-ticked">All services on a single node</li>
+        <li class="p-list__item is-ticked">Opinionated OpenStack</li>
+        <li class="p-list__item is-ticked">Straightforward installation</li>
+        <li class="p-list__item is-ticked">OpenStack “on-rails</li>
       </ul>
-      <p>This is the recommended approach if you want to try OpenStack on your workstation or use it for development purposes. You will be able to perform all basic operations (e.g. upload images, create networks, launch instances, etc.) on a fully functional OpenStack. It takes just a few minutes to install it.</p>
-      <p><a href="#single-node-deployment">Deploy OpenStack on a single node&nbsp;&rsaquo;</a></p>
+      <p><a href="#single-node-deployment">Single-node installation instructions &rsaquo; Deploy OpenStack on a single node&nbsp;&rsaquo;</a></p>
     </div>
     <div class="col-4 p-card">
-      <h3>Multi&dash;node deployment</h3>
+      <h3 class="p-card__title">Multi&dash;node deployment</h3>
+      <p class="p-card__content">Suitable for micro cloud, edge cloud and IoT use cases:</p>
       <ul class="p-list">
-        <li class="p-list__item is-ticked">Uses MicroStack &ndash; snap&dash;based
-          installation</li>
-          <li class="p-list__item is-ticked">At least two machines needed</li>
-          <li class="p-list__item is-ticked">16GB RAM or more each</li>
-          <li class="p-list__item is-ticked">Multi-core processor each</li>
-          <li class="p-list__item is-ticked">50GB of free disk space each</li>
-          <li class="p-list__item is-ticked">Ubuntu 18.04 LTS or later</li>
-          <li class="p-list__item is-ticked">Includes all key OpenStack components</li>
-          <li class="p-list__item is-ticked">Fully compatible with the upstream</li>
-          <li class="p-list__item is-ticked">Supports development iteration</li>
-        </ul>
-        <p>If you are planning to deploy an edge cloud or looking for a solution for the IoT, this approach is the way to go. Similar to the single-node deployment instructions, this method is based on MicroStack which provides access to OpenStack base services and allows basic operations.</p>
+        <li class="p-list__item is-ticked">At least two physical machines needed</li>
+        <li class="p-list__item is-ticked">Uses MicroStack &ndash; OpenStack in a snap</li>
+        <li class="p-list__item is-ticked">Core services included</li>
+        <li class="p-list__item is-ticked">One control node, many compute/storage nodes</li>
+        <li class="p-list__item is-ticked">Opinionated OpenStack</li>
+        <li class="p-list__item is-ticked">Straightforward installation</li>
+        <li class="p-list__item is-ticked">OpenStack “on-rails</li>
+      </ul>
         <p><a href="#multi-node-deployment">Deploy OpenStack on multiple nodes&nbsp;&rsaquo;</a></p>
```

Comment:
```suggestion
        <p><a href="#multi-node-deployment">Multi-node installation instructions&nbsp;&rsaquo;</a></p>
```
Doesn't match copydoc